src/Migrations/M013RemoveDuplicatedOptions.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Remove duplicated Planet 4 features comparing to options.
+ */
+class M013RemoveDuplicatedOptions extends MigrationScript {
+	/**
+	 * Perform the actual migration.
+	 *
+	 * @param MigrationRecord $record Information on the execution, can be used to add logs.
+	 *
+	 * @return void
+	 */
+	protected static function execute( MigrationRecord $record ): void {
+		$features = get_option( 'planet4_features' );
+		$options  = get_option( 'planet4_options' );
+
+		foreach ( array_keys( $features ) as $feature ) {
+			if ( array_key_exists( $feature, $options ) ) {
+				unset( $features[ $feature ] );
+			}
+		}
+
+		update_option( 'planet4_features', $features );
```

Comment:
Hi Dan, Here the code will remove the duplicate value from `planet4_features` instead of `planet4_options`.
And from the ticket description we need to remove duplicates from `planet4_options`

> Remove duplicated values from planet4_options

Reply:
Oh yes! You're completely right